ql/src/java/org/apache/hadoop/hive/ql/plan/ImportTableDesc.java

```diff
@@ -365,4 +366,8 @@ public void setReplWriteId(Long replWriteId) {
       this.createTblDesc.setReplWriteId(replWriteId);
     }
   }
+
+  public Path getTableLocationPath() {
```

Comment:
Shall rename the method to getSourceTblLocation(). Also, table member in the class can be renamed to sourceTable to be more readable.

Reply:
None of the get* methods in ImportTableDesc class are using "source" or "target" tags in their names. I don't see why this API should be an exception.

Instead of adding "source" tag on the source table, ImportTableDesc is using "create" tag on the target table. That looks sufficient to differentiate between target and source. I agree that it would have been more readable to tag the members as "source" and "target" explicity to start with, but if we do it now it's going to be a very large patch since "table" is used at many places in this file. So, I am inclined to leave the status quo as is. We may address it in separate JIRA, but I don't see much need to do so for external table work.

Reply:
The simple rule is methods without "source" prefix means referring to Table/View desc and with prefix means referring to source table. This is the only method referring to source table and it make sense to rename it to be clear.

Reply:
Ok, added "Source" in the name.